codegen/scripts/data.py

```diff
@@ -0,0 +1,139 @@
+"""Parsing and data-related functions"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+import sys
+import os
+from collections import defaultdict, namedtuple
+
+from constants import NUM_CAN_DEVICES, NUM_CAN_MESSAGES, NUM_FIELDS, NUM_DLC_BYTES  # pylint: disable=unused-import
+from google.protobuf import text_format
+
+import can_pb2  # pylint: disable=import-error,wrong-import-position
+
+import validator
+sys.path.append(
+    os.path.abspath(
+        os.path.dirname(os.path.realpath(__file__)) + '/../genfiles'))
```

Comment:
It looks like you or autopep8 moved the `import can_pb2` below the `sys.path.append` construction, which is causing `make codegen` to fail on my machine - can you move it back?